userdocs/profiles.dev/docs/tutorial-basics/install-a-profile.md

```diff
@@ -13,10 +13,9 @@ Please refer back to the [set up docs](/docs/tutorial-basics/setup) if not.
 
 To install a profile, we use `pctl`.
 
-With the following command `pctl` will generate a set of manifests for each profile artifact,
-commit those manifests to a branch in your GitOps repo, push that branch and open a
-PR on your repo to merge the changes.
-Your GitOps repo is the one you synced to Flux in your cluster in the
+With the following command `pctl` will generate a set of manifests for each profile artifact.
+Commit those manifests to a branch in your GitOps repo, push that branch and open a PR to merge the changes.
+Your GitOps repository is the one you synced to Flux in your cluster in the
```

Comment:
This (`Commit`) sounds like an instruction, as in the user should be doing this, where the command will be doing it for them. Maybe split it into bullets?
```suggestion
With the following command `pctl` will:
- generate a set of manifests for each profile artifact
- commit those manifests to a branch in your GitOps repo
- push that branch and
- open a PR to merge the changes

Your GitOps repository is the one you synced to Flux in your cluster in the
```